applications/coreutils/basename.cpp

```diff
@@ -0,0 +1,53 @@
+#include <libsystem/core/CString.h>
+#include <libsystem/io/Stream.h>
+
+/*
+    Implementation of basename shell command.
```

Comment:
We know what is it, no need of all those comments, 
The code speaks for itself

Reply:
Will remove them!